Cartfile

```diff
@@ -1,4 +1,4 @@
 binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon.json" ~> 22.1.0
 binary "https://api.mapbox.com/downloads/v2/carthage/mobile-navigation-native/MapboxNavigationNative.xcframework.json" ~> 111.0
-github "mapbox/mapbox-directions-swift" ~> 2.6
+github "mapbox/mapbox-directions-swift" == 2.7.1
```

Comment:
Since `up to next minor` is not supported by Carthage to prevent build issues related to dropping iOS 10/11 support Mapbox Directions is pinned to specific version.